Jenkinsfile

```diff
@@ -60,6 +65,28 @@ pipeline {
                         }
                     }
                 }
+                stage('Build Invocation image'){
+                    agent {
+                        label 'ubuntu-1604-aufs-edge'
+                    }
+                    steps {
+                        dir('src/github.com/docker/app') {
+                            checkout scm
+                            sh 'make -f docker.Makefile BUILD_TAG=$BUILD_TAG save-invocation-image'
+                            dir('_build') {
+                                stash name: 'invocation-image', includes: 'invocation-image.tar'
+                            }
+                        }
+                    }
+                    post {
+                        always {
+                            dir('src/github.com/docker/app') {
```

Comment:
nit: I don't think the `docker rmi` cares about `cwd` so you could drop this.

Reply:
😭 I don't want to touch this anymore

Reply:
👍